test/json_texture_test.js

```diff
@@ -0,0 +1,28 @@
+var assert = require('assert');
+var configUtils = require('./utils/config');
+var testUtils = require('./utils/test');
+var fs = require('fs');
+var pkg = require('../package.json');
+
+describe('A hash map of texture info, similar to TexturePacker', function () {
+  testUtils.setInfo(configUtils.multipleSprites);
+
+  function assertValidJson() {
+    it('is valid JSON', function () {
+      var result = this.result;
+      assert.doesNotThrow(function () {
+        JSON.parse(result);
+      });
+    });
+  }
+
+  describe('processed by `spritesheet-templates` into JSON', function () {
+    testUtils.runTemplater({format: 'json_texture'});
+
+    var expected = fs.readFileSync(__dirname + '/expected_files/json_texture.json', 'utf8');
+    expected = expected.replace('__VERSION__', pkg.version);
+    testUtils.assertOutputMatchesString(expected);
```

Comment:
I find it weird that I never moved to `deepEqual` comparison for JSON files. However, this current setup is always reading the expected file outside of `mocha's` context (e.g. not in a `before`/`after`/`it`). We should consider removing the test utility and placing it inline for this test:

```js
testUtils.runTemplater({format: 'json_texture'});

it('matches as expected', function () {
  var expected = fs.readFileSync(__dirname + '/expected_files/json_texture.json', 'utf8');
  expected = expected.replace('__VERSION__', pkg.version);
  assert.strictEqual(this,result, expected);
});
```